flex-checks/src/test/resources/checks/NonEmptyCaseWithoutBreak.as

```diff
@@ -15,7 +26,15 @@ function foo() {
     case 0:
     case 1: // NOK
       doSomething();
+    case 2: // NOK
+      if (true) {
+        break;
+      }
+    case 3: // NOK
+      break;
+      doSomething();
     default: // NOK
```

Comment:
@vilchik-elena Should we change this behavior? According to [RSPEC-128](http://jira.sonarsource.com/browse/RSPEC-128), "For the last case, use of break statement is optional".